src/transformers/models/whisper/modeling_whisper.py

```diff
@@ -81,6 +81,33 @@ def _make_causal_mask(input_ids_shape: torch.Size, dtype: torch.dtype, past_key_
         mask = torch.cat([torch.zeros(tgt_len, past_key_values_length, dtype=dtype), mask], dim=-1)
     return mask[None, None, :, :].expand(bsz, 1, tgt_len, tgt_len + past_key_values_length)
 
+# To only condition from certain indexes
```

Comment:
Can remove this comment here and place it below in the docstring for the function!